scripts/fetch-proto.sh

```diff
@@ -3,7 +3,7 @@ OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')
 ARCH=$(uname -m)
 ORG_NAME="dapr"
 REPO_NAME="dapr"
-BRANCH_NAME="v1.9.0"
+BRANCH_NAME="master" # TODO: change to release branch
```

Comment:
Left TODO?

Reply:
This is to be done once Dapr 1.10 is released, this PR can't be merged until then.